avocado/utils/process.py

```diff
@@ -98,6 +98,18 @@ def __str__(self):
             return "CmdError"
 
 
+def sudo_available():
+    """
+    :return: True when sudo is available (or is root)
+    """
+    if os.getpid() == 0:
+        return True
+    elif system_output("id -u", ignore_status=True, sudo=True).strip() == "0":
```

Comment:
It'd feel unnatural to me to call python to supple coreutils program. If we ever support windows, then we might try it, but we can as well call this or windows equivalent, because running python (50m ram) or id (0.0m ram) is a big difference. (my issue is not with the memory usage, that is only an example, my point is running a python as sudo just to get uid)